crates/subspace-runtime/src/lib.rs

```diff
@@ -495,6 +498,110 @@ impl pallet_preimage::Config for Runtime {
     type WeightInfo = pallet_preimage::weights::SubstrateWeight<Runtime>;
 }
 
+type HasherOutputFor<T> = <T as sp_core::Hasher>::Out;
+
+pub struct DepthLimitedPreimage;
```

Comment:
I'm not sure why we need this here?

Why do we need a DepthLimitedPreImage?

If we want to have such a usecase, we should ideally pull this change to upstream first and then bring this in our fork as upstream will also be affected with this issue,

Reply:
yeah, lets document this instead. Thanks!